src/pages/settings/AddSecondaryLoginPage.js

```diff
@@ -129,30 +129,19 @@ class AddSecondaryLoginPage extends Component {
                     <View style={styles.mb6}>
                         <TextInput
                             label={this.props.translate('common.password')}
-                            value={this.state.password}
-                            onChangeText={password => this.setState({password})}
+                            inputID="password"
                             secureTextEntry
                             autoCompleteType="password"
                             textContentType="password"
-                            onSubmitEditing={this.submitForm}
+                            onSubmitEditing={this.onSubmit}
                         />
                     </View>
                     {!_.isEmpty(this.props.user.error) && (
                         <Text style={styles.formError}>
                             {this.props.user.error}
                         </Text>
                     )}
-                </ScrollView>
-                <FixedFooter style={[styles.flexGrow0]}>
-                    <Button
-                        success
-                        isDisabled={this.validateForm()}
-                        isLoading={this.props.user.loading}
```

Comment:
Since the loading prop is at the `user` key we won't see the loading indicator at the button. Please address it.